composer.json

```diff
@@ -33,9 +39,5 @@
             "@composer validate --no-check-all --strict",
             "@tests"
         ]
-    },
-    "config": {
-        "sort-packages": true,
```

Comment:
sort packages mohlo zůstat, jen ten HTTPS jsem myslel pryč. Ale je to asi jen detail.

Reply:
revert